render/helpers.go

```diff
@@ -3,20 +3,44 @@ package render
 import (
 	"fmt"
 	"html/template"
+	"os"
 	"reflect"
 	"regexp"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/BurntSushi/toml"
+	"github.com/ONSdigital/dp-frontend-renderer/config"
 	"github.com/ONSdigital/go-ns/log"
 	"github.com/c2h5oh/datasize"
 	"github.com/gosimple/slug"
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"golang.org/x/text/language"
 	blackfriday "gopkg.in/russross/blackfriday.v2"
 )
 
 const legacyDatasetURIFormat = "/file?uri=%s/%s"
 
+var bundle = InitLocaleBundle()
+
+// InitLocaleBundle is used to initialise the locale bundle
+func InitLocaleBundle() *i18n.Bundle {
+	bundle := i18n.NewBundle(language.English)
+	bundle.RegisterUnmarshalFunc("toml", toml.Unmarshal)
+	workingDir, err := os.Getwd()
+	if err != nil {
+		log.Error(err, nil)
```

Comment:
Do we want to carry on if this fails? Should we return an error? or should it log and return at this point?

Reply:
Good point, I have added an error log and return when this happens now.